### PR TITLE
[UILISTS-137] Added shortcut keys custom list

### PR DESCRIPTION
**Pull request by SergeYvas**
Implementation of [UILISTS-137](https://folio-org.atlassian.net/browse/UILISTS-137)

Added shortcut keys to custom list

Screenshot 2024-08-01 at 16 33 06

### CODE REVIEW

**Review by vashjs — APPROVED** [review]
Please update CHANGELOG.MD if needed.

File: src/keyboard-shortcuts.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+export const commandsGeneral = [
+  shortcutItem('commands-label.create', 'Option + n'),
+  shortcutItem('commands-label.edit', 'cmd + Option + e'),
+  shortcutItem('commands-label.save', 'cmd + s'),
+  shortcutItem('commands-label.duplicate', 'Option + C'),
+  shortcutItem('commands-label.toggle-lists-detail-accordion', 'spacebar'),
+  shortcutItem('commands-label.expand-list-detail-accordions', 'cmd  + Option + b'),
+  shortcutItem('commands-label.collapse-list-detail-accordions', 'cmd  + Option + g'),
+  shortcutItem('commands-label.go-to-filter-pane', 'cmd + s'),
```
Comment (mweaver-ebsco) [inline src/keyboard-shortcuts.tsx:20]: Do these shortcuts follow the key name conventions used in other parts of folio (e.g., cmd vs command vs Command, opt vs option vs Option, n vs N, etc)

I'm assuming this is okay, but just want to double check :)

File: translations/ui-lists/en.json [hunk 2]
```diff
@@ -170,5 +170,19 @@
...
+
+  "commands-label.create": "Create a new list",
+  "commands-label.edit": "Edit a list",
+  "commands-label.save": "Save a list" ,
+  "commands-label.duplicate": "Duplicate a list",
+  "commands-label.toggle-lists-detail-accordion": "Expand or collapse a Lists detail accordion",
+  "commands-label.expand-list-detail-accordions": "Expand all List detail accordions",
+  "commands-label.collapse-list-detail-accordions": "Collapse all List detail accordions",
+  "commands-label.go-to-filter-pane": "Go to Lists Filter pane",
+  "commands-label.close-modal-pop-up": "Close a modal or pop-up",
```
Comment (mweaver-ebsco) [inline translations/ui-lists/en.json:183]: Modal probably isn't a term that is familiar to most non-devs. Is there another word we could use?

Comment (mweaver-ebsco) [reply translations/ui-lists/en.json:183]: I looked in other folio apps, and it looks like they word it the same, so I'd leave it as is :)

File: src/keyboard-shortcuts.tsx (hunk 1 above)
Comment (mweaver-ebsco) [reply src/keyboard-shortcuts.tsx]: I looked around a bit, and it looks like other apps capitalize the first letter in "cmd" and "spacebar" along with the single letters (N, E, S, etc)

File: src/keyboard-shortcuts.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+export const commandsGeneral = [
+  shortcutItem('commands-label.create', 'Option + n'),
+  shortcutItem('commands-label.edit', 'Cmd + Option + e'),
+  shortcutItem('commands-label.save', 'Cmd + s'),
+  shortcutItem('commands-label.duplicate', 'Option + C'),
+  shortcutItem('commands-label.toggle-lists-detail-accordion', 'spacebar'),
+  shortcutItem('commands-label.expand-list-detail-accordions', 'Cmd  + Option + b'),
+  shortcutItem('commands-label.collapse-list-detail-accordions', 'Cmd  + Option + g'),
+  shortcutItem('commands-label.go-to-filter-pane', 'Cmd + s'),
+];
```
Comment (mweaver-ebsco) [inline src/keyboard-shortcuts.tsx:21]: Do we need to do anything special for non-Mac users?

Comment (SergeYvas) [reply src/keyboard-shortcuts.tsx:21]: Inside the modal component, Mod and Cmd will be replaced with Alt and Option automatically
https://github.com/folio-org/stripes-components/blob/d3cf1280b26e2d313441d6069be2eec44215da9b/lib/KeyboardShortcutsModal/KeyboardShortcutsModal.js#L34

**Review by mweaver-ebsco — APPROVED** [review]

**Pull request merged**